osbs/cli/main.py

```diff
@@ -684,13 +684,13 @@ def main():
         logger.error("Configuration error: %s", ex.message)
         return -1
 
-    is_verbose = os_conf.get_verbosity()
+    is_verbose = os_conf.get_verbosity() or args.verbose
```

Comment:
`args.verbose` is already handled by osbs.conf.Configuration (not very obvious), so I removed that, and limited the change to prioritize the `--quiet` flag correctly.